src/lib/i18n/texts/de.json

```diff
@@ -66,22 +66,22 @@
       "losses": "Niederlagen",
```

Comment:
well looking here: https://wiki.guildwars2.com/wiki/Structured_PvP

"One or more players of the enemy team disconnect from the match: You will get less rewarded for winning."

seems about right. is the word in the brackets needed for context to make sense?